website/common/locales/en/gear.json

```diff
@@ -101,7 +102,7 @@
   "weaponSpecialSkiText": "Ski-sassin Pole",
   "weaponSpecialSkiNotes": "A weapon capable of destroying hordes of enemies! It also helps the user make very nice parallel turns. Increases Strength by <%= str %>. Limited Edition 2013-2014 Winter Gear.",
   "weaponSpecialCandycaneText": "Candy Cane Staff",
-  "weaponSpecialCandycaneNotes": "A powerful mage's staff. Powerfully DELICIOUS, we mean! Two-handed weapon. Increases Intelligence by <%= int %> and Perception by <%= per %>. Limited Edition 2013-2014 Winter Gear.",
```

Comment:
This is only going to work on the website but will remove completely any info about the item being two-handled in mobile apps for example